web/docs/Frontend Starters/Gatsby/Gatsby + WordPress/add-ons.md

```diff
@@ -0,0 +1,87 @@
+---
+id: 'gatsby-wordpress-add-ons'
+title: 'Add-ons'
+slug: '/frontend-starters/nextjs/gatsby/gatsby-wordpress/add-ons'
+sidebar_position: 5
+---
+
+## Before You Begin
+
+- Install and activate the
+  [WP Advanced Custom Fields Plugin](https://wordpress.org/plugins/advanced-custom-fields/).
+
+- Familiarize yourself with the
+  [Create Pantheon Decoupled Kit CLI](https://www.npmjs.com/package/create-pantheon-decoupled-kit/).
+  Any package manager can be used with the CLI **[npm, pnpm, yarn]**.
+
+## Advanced Custom Fields WordPress Plugin
+
+The Advanced Custom Fields WordPress plugin, or ACF, turns WordPress sites into
+a fully-fledged content management system by giving you all the tools to do more
+with your data.
```

Comment:
Two things:
1. If you take text directly from a source on the web, add an attribution to that source.
2. I want to avoid explaining what ACF is in our docs. I would expect someone reading this to already be familiar with ACF, and if not we can link to their docs so the user can learn about it before coming back here and integrating ACF with our starter via the add-on. So in that case, I would prefer a note in "Before You Begin" that explains "You should be familiar with WordPress Advanced Custom Fields" with a link to the docs. I don't think we should explain what it is here.

This section should explain what our add-on does, but we should avoid explaining ACF.